apps/bfd-server/bfd-server-war/src/main/config/server-start.sh

```diff
@@ -193,6 +193,7 @@ BFD_PORT="${serverPortHttps}" \
 	"-DbfdServer.db.username=" \
 	"-DbfdServer.db.password=" \
 	"-DbfdServer.db.schema.apply=true" \
+	"-DbfdServer.include.fake.drug.code=true" \
```

Comment:
I know none of the other ones are comments but I think we should write a comment in here about why we are passing this, that it only applies to testing, etc

Reply:
Done